docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+db:
+    restart: always
+    image: mysql:5.6
+    ports:
+        - "3306:3306"
+    environment:
+        MYSQL_ROOT_PASSWORD: changeme
+        #the values below should match the credentials used in 
+        #instance/local_settings.py
+        MYSQL_USER: nbsap
+        MYSQL_PASSWORD: nbsap
+        MYSQL_DATABASE: nbsap
+web:
+    build: .
+    command:
+        python instance/manage.py runserver 0.0.0.0:8000
+    volumes:
+        - .:/nbsap
+    ports:
+        - "8000:8000"
+    links:
+        - db
+    environment:
+        - DEBUG=True
```

Comment:
In https://github.com/eea/eea.docker.nbsap you should put two docker-compose files:

- this file, but renamed `docker-compose.dev.yml`, to be used as a development setup
- a new one, named `docker-compose.yml` with containers setup for the instances on calamar (the country instances + demo + ec)